runtime/src/bank.rs

```diff
@@ -65,6 +65,8 @@ use crate::{
 use byteorder::{ByteOrder, LittleEndian};
 use itertools::Itertools;
 use log::*;
+use rayon::iter::IntoParallelIterator;
+use rayon::iter::ParallelIterator;
```

Comment:
I think we're using the combined `use` pattern here. Do you mind combining the `rayon` entries to keep it consistent?